Makefile

```diff
@@ -156,6 +158,17 @@ srpm:
 	RELEASE_PRE=${RELEASE_PRE} ./packaging/rpm/make-rpm.sh local
 .PHONY: srpm
 
+image-build-configure:
+	./scripts/image-builder/configure.sh
+.PHONY: image-build-configure
+
+image-build-iso: rpm 
```

Comment:
I didn't choose the same filename for $(PULLSECRET) as the value on line 40, and the build didn't fail when that file was not present. We should make build.sh report that error.

Reply:
Thanks @dhellmann, we collided on adding a check for the pull secret file. I'll back out my change.